docs/install/update-harvester-configuration.md

```diff
@@ -47,6 +47,8 @@ If you upgrade from a version before `v1.1.2`, the `cloud-init` file in examples
 
 ### Configuration persistence
 
+The persistent filename previously was `/oem/99_custom.yaml`. It is now `/oem/90_custom.yaml` in Harvester v1.1.2 and later versions. That is because we use a newer `elemental` version. This file is created by `elemental` with installation time. If you upgrade from a version before `v1.1.2`, this file will be `/oem/99_custom.yaml`. We do not rename this file with an upgrade because we do not want the user to get confused. If you align the new mechanism, you can manually rename this file to `/oem/90_custom.yaml`.
```

Comment:
```suggestion
Beginning with v1.1.2, the persistent name of the cloud-init file is `/oem/90_custom.yaml`. Harvester now uses a newer version of Elemental, which creates the file during installation.

When upgrading from an earlier version to `v1.1.2` or later, Harvester retains the old file name (`/oem/99_custom.yaml`) to avoid confusion. You can manually rename the file to `/oem/90_custom.yaml` if necessary.
```